### ISSUE TITLE
`dist-build` process doesn't keep the readme.txt

### ISSUE THREAD

**Issue by bacoords** [understrap/understrap#1528]
When running `npm run dist-build`, we trim out many source files, assets. etc, but we also remove the `readme.txt` which is required for the WordPress theme directory. Any concerns about keeping `readme.txt` in the dist/ folder?

**Comment by bacoords** [comment]
Also it should strip out the `'.git'` folder

**Comment by bacoords** [comment]
And I've been messing with this: 
```js
const pkg = require('../../package.json')
...
copyDir('./', `./dist/${pkg.name}-${pkg.version}`);
```
which would put your dist version in a subfolder like this: `dist/understrap-child-1.1.0/` or something similar. Makes things a little easier..

**Comment by kirannasim** [comment]
@bacoords This works well. I updated src/build/dist-build.js and tested it

https://github.com/understrap/understrap/pull/1620

**Comment by IanDelMar** [comment]
@bacoords fixed in v1.2.0
